docs/installing-solidity.rst

```diff
@@ -6,6 +6,28 @@
 Installing Solidity
 ###################
 
+Versioning
```

Comment:
I think most people don't care, they only want the latest stable. Browser-solidity provides the latest stable, as do the non-dev ppas.